src/RoleObjects/AccessibilityObject.js

```diff
@@ -177,6 +177,117 @@ export default class AccessibilityObject {
     return this._reactProps.accessKey;
   }
 
+  /**
+   * The translate attribute specifies whether the content of an element should be translated or not.
+   * @access public
+   * @param {String} str - "yes" or "no" Specifies that the content of the element should be translated or not
+   */
+  set translate(str) {
+    this._reactProps.translate = str;
+  }
+
+  /**
+   * The translate attribute specifies whether the content of an element should be translated or not.
+   * @access public
+   * @returns {String} string containing the letter to use for the keyboard shortcut
+   */
+  get translate() {
+    return this._reactProps.translate;
+  }
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @param { boolean } bool
+   */
+  set spellcheck(bool) {
+    this._reactProps.spellCheck = bool;
+  }
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @returns {boolean} boolean: true or false
+   */
+  get spellcheck() {
+    return this._reactProps.spellCheck;
+  }
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @param { boolean } bool true or false
+   */
+  set contenteditable(bool) {
```

Comment:
Please see my comment in your corresponding CAM tester PR (https://github.com/CurriculumAssociates/createjs-accessibility-tester/pull/6) for details, but I think we can skip supporting this attribute.  So, please remove the getter and setter for it.

Reply:
updated